### PR TITLE
Don't restart login activity on orientation changes

### LINKED ISSUE #1368: Login webview content lost after orientation change
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues?q=is%3Aissue+is%3Aclosed) issues to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.
- [X] If the bug only occurs with a certain link, post, image..., I will include the URL.

### App version

5.4.0

### Where did you get the app from

F-Droid

### Android version

10

### Device model

emulator

### First occurred

_No response_

### Steps to reproduce

Open login, randomly click some links

Before rotation

Screen Shot 2023-02-10 at 3 09 19 PM

After rotation
Screen Shot 2023-02-10 at 3 09 27 PM

### Example post, link, markdown...

_No response_

### Expected behaviour

Page should remain unchanged

### Current behaviour

_No response_

### Logs

_No response_

### PR DESCRIPTION
**Pull request by camporter**
Fixes #1368

Keeps the state of the webview in place during orientation and screen changes.

Tested it a bit, but some less common cases might need more testing.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -387,6 +387,7 @@
         </activity>
         <activity
             android:name=".activities.LoginActivity"
+            android:configChanges="orientation|screenLayout|screenSize|layoutDirection"
```
Comment (shbhmnk) [inline app/src/main/AndroidManifest.xml:390]: Can you please verify if screenLayout, screenSize and layoutDirection are necessary for this.

**Review by shbhmnk — APPROVED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (camporter) [reply app/src/main/AndroidManifest.xml:390]: I believe most of these should be useful. For example, if a user switches the display resolution (eg from 1080p to 1440p) while filling out the login form, the login page is reloaded similarly to how changing the display orientation reloads the page and loses the login details.

Comment (shbhmnk) [reply app/src/main/AndroidManifest.xml:390]: That makes sense. Could you update the title and description for the PR to reflect the same details. You can also add them in #1368

**Comment by Docile-Alligator** [comment]
Thank you!

**Pull request merged**